ml-proto/spec/kernel.ml

```diff
@@ -82,9 +82,9 @@ and expr' =
   | Block of expr list                      (* execute in sequence *)
   | Loop of expr                            (* loop header *)
   | Break of var * expr option              (* break to n-th surrounding label *)
-  | Br_if of var * expr option * expr       (* conditional break *)
+  | Break_if of var * expr option * expr    (* conditional break *)
+  | Break_table of var list * expr option * expr  (* indexed break *)
```

Comment:
That was a text-format discussion and I think, in that proposal, the user would see both "break" and "continue" (therefore going the correct direction) so that's a separate topic altogether; here we have a single node that is jumping both forward and backward.